x-pack/plugins/integration_assistant/server/graphs/ecs/constants.ts

```diff
@@ -1703,92 +1751,314 @@ export const ECS_FIELDS: EcsFields = {
+  'email.from.domain': 'Domain of the sender.',
+  'email.from.local': 'Local part of the sender.',
+  'email.from.registered_domain':
+    'The highest registered sender domain, stripped of the subdomain.',
+  'email.from.subdomain': 'Subdomain of the sender.',
+  'email.from.top_level_domain': 'Top level domain of the sender.',
+  'email.local_id': 'Local identifier of the email.',
+  'email.message_id': 'Message ID of the email.',
+  'email.origination_timestamp': 'Time the email was originated.',
+  'email.reply_to.address': 'Email address of the reply-to.',
+  'email.reply_to.domain': 'Domain of the reply-to.',
+  'email.reply_to.local': 'Local part of the reply-to.',
+  'email.reply_to.registered_domain':
+    'The highest registered reply-to domain, stripped of the subdomain.',
+  'email.reply_to.subdomain': 'Subdomain of the reply-to.',
+  'email.reply_to.top_level_domain': 'Top level domain of the reply-to.',
+  'email.sender.address': 'Email address of the sender.',
+  'email.sender.domain': 'Domain of the sender.',
+  'email.sender.local': 'Local part of the sender.',
+  'email.sender.registered_domain':
+    'The highest registered sender domain, stripped of the subdomain.',
+  'email.sender.subdomain': 'Subdomain of the sender.',
+  'email.sender.top_level_domain': 'Top level domain of the sender.',
+  'email.size': 'Size of the email in bytes.',
+  'email.subject': 'Subject of the email.',
+  'email.to.address': 'Email addresses of recipients.',
+  'email.to.domain': 'Domain of recipients.',
+  'email.to.local': 'Local part of the recipients.',
+  'email.to.registered_domain':
+    'The highest registered recipient domain, stripped of the subdomain.',
+  'email.to.subdomain': 'Subdomain of the recipients.',
+  'email.to.top_level_domain': 'Top level domain of the recipients.',
+  'error.code': 'Error code describing the error.',
+  'error.id': 'Unique identifier for the error.',
+  'error.message': 'Error message.',
+  'error.stack_trace': 'Error stack trace.',
+  'error.type': 'Error type.',
   'event.action': 'The action captured by the event.',
-  'event.created': 'Time when the event was first read by an agent or by your pipeline.',
+  'event.category': 'Event category.',
   'event.code': 'Identification code for this event.',
+  'event.created': 'Date/time when the event was first read by an agent, or by your pipeline.',
+  'event.dataset': 'Name of the dataset.',
   'event.duration': 'Duration of the event in nanoseconds.',
-  'event.end':
-    'event.end contains the date when the event ended or when the activity was last observed.',
+  'event.end': 'Date when the event ended.',
+  'event.hash': 'Hash of the event.',
   'event.id': 'Unique ID to describe the event.',
+  'event.ingested': 'Timestamp when an event arrived in the central data store.',
+  'event.kind': 'The kind of event.',
+  'event.module': 'Name of the module this data is coming from.',
+  'event.original': 'Raw text message of entire event.',
```

Comment:
I think we should omit this as well

Reply:
Looks like `event.original` still exists?

Reply:
Oops.. A miss :-)